hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -312,6 +312,7 @@ private ContainerCommandResponseProto dispatchRequest(
     }
     responseProto = handler.handle(msg, container, dispatcherContext);
     if (responseProto != null) {
+      metrics.incContainerOpsMetrics(Type.CreateContainer);
       metrics.incContainerOpsLatencies(cmdType,
```

Comment:
We shouldn't report `CreateContainer` op for "any" command.

The first attempt was fine: increment ops count if `container` is being created inline for other (WriteChunk, PutSmallFile, etc.) commands.  Latency can be updated there, too, we just need to take note of start time, and use it to calculate elapsed time.

Reply:
oh! i got it! ok let me update!